src/main/scala/com/github/ahnfelt/react4s/Component.scala

```diff
@@ -90,7 +90,7 @@ sealed trait ElementOrComponent extends Tag {
 }
 
 /** Wraps a React component written in JavaScript. Example: ```DynamicComponent(js.Dynamic.global.MyJsComponent, js.Dictionary("label" -> "Go!"))``` */
-final case class DynamicComponent(componentClass : Any, props : Any, key : Option[String] = None, ref : Option[Any => Unit] = None) extends ElementOrComponent {
+final case class DynamicComponent(componentClass : Any, props : Any, children : Seq[Tag], key : Option[String] = None, ref : Option[Any => Unit] = None) extends ElementOrComponent {
```

Comment:
Ah, I see. But `props` may actually contain the `children`, `key` and `ref` fields already - that's why `key` and `ref` have default values, and I think it makes sense to give `children : Seq[Tag] = Seq.empty` in the same vein.